Cargo.toml

```diff
@@ -25,7 +25,6 @@ hashbrown = "0.1.8"
 log = "0.4.2"
 rand = "0.6.5"
 rayon = "1.0.0"
-reqwest = "0.9.11"
```

Comment:
Nice! But why do we have any dependencies in this file?

Reply:
tests/rpc.rs

Reply:
nevermind, I can fix it.